action.yml

```diff
@@ -5,7 +5,7 @@ inputs:
     description: 'Relative path to the documentation file'
     required: true
     default: api-contract.yml
-  id:
```

Comment:
Removing the id will generate a GitHub action error here, for those who currently use the action, until they update the call by replacing `id` per `doc`.

I think that it's acceptable as for now we do not version this tool (and we do not have a lot of customers using this action at the moment), but once this is done we should clearly start tagging versions and point the README example and the action example to the latest tag.

WDYT @antlio?

Reply:
another option is to support both `id` and `doc` as optional parameters and only the good one dynamically

Reply:
Also I think that we can also just break the compatibility since the current version (0.1) is broken and make the CI fail anyway :)

Reply:
I agree with the last @moul comment, which join your first assumption @scharrier 😋